### PR TITLE
Close Lumen and add ebay

### PR DESCRIPTION
**Pull request by VinceStargazer**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -351,7 +351,7 @@ And we're back! Use this repo to share and keep track of software, tech, CS, PM,
 | [Crusoe](https://jobs.lever.co/crusoeenergy/16d9dc19-74a5-4c32-9ead-e3e221f4eaca) | San Francisco, CA | University / Software Engineer (Intern) |
 | [Farm Credit Services of America](https://careers.fcsamerica.com/job/16636430/) | Omaha, NE | Web Software Systems Engineer (Intern Summer 2023) |
 | Raytheon | Varies by Role | [UP 2023 Software Engineer Internships](https://careers.rtx.com/global/en/job/RAYTGLOBAL01567022EXTERNALENGLOBAL/UP-2023-Software-Engineer-Internships), [2023 Software Engineer Summer Intern](https://careers.rtx.com/global/en/job/RAYTGLOBAL01569607EXTERNALENGLOBAL/2023-Software-Engineer-Summer-Intern) (US Citizenship Required for Both) |
-| [Lumen](https://jobs.lumen.com/global/en/university-entry-level) | Remote | [Intern - Software Engineer - Summer 2023 Nationwide / Remote](https://jobs.lumen.com/global/en/job/317271/Intern-Software-Engineer-Summer-2023), [Intern - Data Scientist](https://jobs.lumen.com/global/en/job/318304/Intern-Data-Scientist-Summer-2023-Nationwide-Remote?job) |
```
Comment (ewang2002) [inline README.md]: Are you sure Lumen is closed? I was able to access the application just fine.

**Review by ewang2002 — COMMENTED** [review]
You have some merge conflicts apparently. Can you try to resolve them (try to sync your fork)? 

I — or another maintainer — can do it if you aren't able to later.

Also, see comment on Lumen.

File: README.md (hunk 1 above)
Comment (owini) [reply README.md]: Good catch! Lumen's still open.

**Comment by owini** [comment]
Updated and added eBay. Thanks!

File: README.md (hunk 1 above)
Comment (VinceStargazer) [reply README.md]: Thanks for pointing it out! The page told me the job posting was gone several minutes ago but now it was back. Maybe some system glitch :)

**Pull request merged**